src/main/java/com/dumbdogdiner/stickyapi/common/util/NumberUtil.java

```diff
@@ -45,27 +45,29 @@ private NumberUtil() {
     public static boolean isNumeric(@NotNull String string) {
         int size = string.length();
         for (int i = 0; i < size; i++) {
-            if (Character.isDigit(string.charAt(i)) == false) {
+            if (!Character.isDigit(string.charAt(i))) {
                 return false;
             }
         }
         return true;
     }
 
+    // TODO the percent stuff can be rewritten in a more compact manor
+
     /**
      * Get a number as the percentage of another.
      * 
      * @param x      The number who's percentage of the total this method will
      *               return
      * @param total  The total
-     * @param string If this should return as a string with `%` appended to the end
+     * @param withSymbol If this should return as a string with `%` appended to the end
      * @return {@link Double}
      */
-    public static String getPercentage(@NotNull double x, @NotNull double total, @NotNull Boolean string) {
-        var percent = (x / total);
-        StringBuilder sb = new StringBuilder();
-        if (string) {
-            sb.append((percent * 100) + "%");
+    public static @NotNull String getPercentage(@NotNull double x, @NotNull double total, boolean withSymbol) {
```

Comment:
Worth considering moving to primitives like you have here where we can - seems sensible!